frontend/src/components/regions/DueTimeFormModal/Presentation.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  TextField,
+  Button,
+  Grid,
+  Paper,
+} from "@material-ui/core";
+import Modal from '../../widgets/Modal';
+
+const useStyles = makeStyles({
+
+    dueTimeButton: {
+        marginTop: "0.3rem",
+        marginRight: "0.3rem",
+    },
+});
+
+const DueTimeFormModal = ({ card, closeModal }) => {
+
+  const {dueTime} = card || {};
```

Comment:
under what circumstances would you want the due time to be an object? Shouldn't it always be a string here? 
Does your story work if there is no card passed into the presentation component?